product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/01_private_endpoint.mdx

```diff
@@ -0,0 +1,148 @@
+
+  To connect to the PG service, create an Azure Private Endpoint in your VM's virtual network. After you have created the private endpoint, you can use its private IP address to access the PG service.
+  
+  1. On the upper-left side of the screen in the portal, select **Create a resource > Networking > Private Link** or in the search box enter Private Link.
+  
+  1. Select **Create**.
+
+  1. In Private Link Center, select **Private endpoints** in the left-hand menu.
+
+  1. In Private endpoints, click **Create**.
+
+  1. Enter "pg-service-private-link" as the name for the Private Link Service and provide additional details as shown in the following screenshot.
+
+   ![](../images/image12.png)
+
+  1. Connect the private endpoint to the EDB Cloud private link service that we created by entering its *alias*.
+
+   ![](../images/image17.png)
+
+  1. In the **Configuration** page, enter the VM's Virtual Network `vnet-client`.
+
+  1. Select **Review + Create**.
+
+  7. Select **Create**.
+   
+  8. You might have to approve the connection request from the `pg-service-private-link` owner, if the private endpoint's status appears as **Pending** for a long period. 
+   
+   Switch to the EDB Cloud subscription and from the **Pending connections** page in **Private Link Center** approve the connection request.
+
+  ![](../images/image3.png)
+
+
+11. Now that you have successfully built a tunnel between your VM's virtual network and the EDB Cloud cluster network, access the private PG service by using the private endpoint.
+   
+   Get the private endpoint's private IP address using the following scripts:
+
+ ```
+  $ NICID=$(az network private-endpoint show -n pl-private-pg-service
+  -g rg-client --query "networkInterfaces[0].id" -o tsv)
+  $ az network nic show -n ${NICID##*/} -g rg-client --query
+  "ipConfigurations[0].privateIpAddress" -o tsv
+  100.64.111.5
+ ``` 
+
+12. From the VM `vm-client`, access the PG service by using the private IP address.
+
+ ![](../images/image20.png)
+
+#### Step 3: Create Azure Private DNS Zone for the Private Endpoint
+
+EDB recommends using an [Azure Private DNS Zone](https://docs.microsoft.com/en-us/azure/dns/private-dns-privatednszone) with the Private Endpoint to establish a secure connection with the PG service. Provide a private domain name identical to the EDB Cloud PG service's default domain name, i.e. `p-c4ia6jleig40jngmac40.brcxzr08qr7rbei1.edbcloud.io`. 
+     
+Azure's DNS resolving mechanism ensures that all requests to the domain name from `vnet-client` are resolved to the private endpoint's private IP address `pl-private-pg-service` instead of the PG service's IP address.
+
+1. In the Azure portal search bar, enter "Private DNS Zones" in the search text box and press Enter.
+ 
+1. Click **Private DNS zone**.
+
+1. Click **Create private DNS zone**.
+
+1. Create a private DNS zone using your organization-level domain
```

Comment:
We should note what the "organization-level domain" is, and make sure it's clear that (1) this step needs to be done once per VNet, even if you have multiple clusters you're connecting to, and (2) the organization-level domain is unique to your EDB Cloud organization.